### PR TITLE
Dịch "momentum_vn" - Phần 5

### LINKED ISSUE #1907: Dịch "momentum_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_optimization/momentum_vn.md#L257-L374

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by tranyenthy**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1907 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_optimization/momentum_vn.md [hunk 1]
```diff
@@ -291,29 +295,31 @@ d2l.plt.legend();
...
 <!--
 Compared with (minibatch) SGD the momentum method needs to maintain a set of  auxiliary variables, i.e., velocity.
 It has the same shape as the gradients (and variables of the optimization problem).
 In the implementation below we call these variables `states`.
 -->
 
-*dịch đoạn phía trên*
-
+So với SGD (minibatch), phương pháp động lượng cần duy trì một tập hợp các biến phụ trợ, tức là vận tốc.
+Nó có hình dạng giống như độ dốc (và các biến của vấn đề tối ưu hóa).
```
Comment (cuongvng) [inline chapter_optimization/momentum_vn.md:321]:
```suggestion
Nó có kích thước giống gradient (và các biến của vấn đề tối ưu hóa).
```

File: chapter_optimization/momentum_vn.md [hunk 2]
```diff
@@ -291,29 +295,31 @@ d2l.plt.legend();
...
+Trong phần thực hiện bên dưới, chúng ta gọi các biến này là `states` (các trạng thái).
```
Comment (cuongvng) [inline chapter_optimization/momentum_vn.md:322]:
```suggestion
Trong phần lập trình bên dưới, chúng ta gọi các biến này là `states` (các trạng thái).
```

File: chapter_optimization/momentum_vn.md [hunk 3]
```diff
@@ -351,7 +357,8 @@ When we increase the momentum hyperparameter `momentum` to 0.9, it amounts to a
 We reduce the learning rate slightly to $0.01$ to keep matters under control.
 -->
 
-*dịch đoạn phía trên*
+Khi chúng tôi tăng siêu tham số động lượng `momentum` lên 0,9, nó sẽ có kích thước mẫu hiệu quả lớn hơn đáng kể là $\frac{1}{1 - 0.9} = 10$.
```
Comment (cuongvng) [inline chapter_optimization/momentum_vn.md:360]:
```suggestion
Khi tăng siêu tham số động lượng `momentum` lên 0,9, kích thước mẫu hiệu quả sẽ tăng lên đáng kể thành $\frac{1}{1 - 0,9} = 10$.
```

File: chapter_optimization/momentum_vn.md [hunk 4]
```diff
@@ -351,7 +357,8 @@ When we increase the momentum hyperparameter `momentum` to 0.9, it amounts to a
...
+Chúng tôi giảm tỷ lệ học tập xuống còn $0.01$ để kiểm soát các vấn đề.
```
Comment (cuongvng) [inline chapter_optimization/momentum_vn.md:361]:
```suggestion
Chúng tôi giảm tỷ lệ học tập xuống còn $0,01$ để kiểm soát các vấn đề.
```

File: chapter_optimization/momentum_vn.md [hunk 5]
```diff
@@ -364,7 +371,9 @@ Reducing the learning rate further addresses any issue of non-smooth optimizatio
 Setting it to $0.005$ yields good convergence properties.
 -->
 
-*dịch đoạn phía trên*
+Giảm tỷ lệ học tập tiếp tục giải quyết bất kỳ vấn đề tối ưu hóa không trơn tru.
```
Comment (cuongvng) [inline chapter_optimization/momentum_vn.md:374]:
```suggestion
Tiếp tục giảm tốc độ học sẽ giải quyết bất kỳ vấn đề nào của bài toán tối ưu không trơn tru.
```

File: chapter_optimization/momentum_vn.md [hunk 6]
```diff
@@ -260,7 +260,7 @@ Let us look at what leaky averaging actually does to the updates.
 ### Effective Sample Weight
 -->
 
-### *dịch tiêu đề phía trên*
+### Trọng số mẫu hiệu quả
```
Comment (lkhphuc) [inline chapter_optimization/momentum_vn.md:263]:
```suggestion
### Trọng số mẫu thực sự
```
`effective` ở đây mang nghĩa `actual == thực sự/ thực tế` ấy, giống cụm `the effective ruler == người cai trị thực sự`

File: chapter_optimization/momentum_vn.md [hunk 7]
```diff
@@ -291,29 +295,31 @@ d2l.plt.legend();
...
-## *dịch tiêu đề phía trên*
+## Các thực nghiệm
 
 <!--
 Let us see how momentum works in practice, i.e., when used within the context of a proper optimizer.
 For this we need a somewhat more scalable implementation.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta hãy xem động lượng hoạt động như thế nào trong thực tế, tức là, khi được sử dụng trong bối cảnh của trình tối ưu hóa phù hợp.
```
[the remark this answers is not in the document]
Comment (lkhphuc) [reply chapter_optimization/momentum_vn.md:305]:
```suggestion
Chúng ta hãy xem động lượng hoạt động như thế nào trong thực tế, tức là, khi được sử dụng trong bối cảnh của một bộ tối ưu hóa đích thực.
```

File: chapter_optimization/momentum_vn.md (hunk 1 above)
Comment (lkhphuc) [reply chapter_optimization/momentum_vn.md:321]:
```suggestion
Nó có kích thước giống gradient (và các biến khác trong bài toán tối ưu hóa).
```

File: chapter_optimization/momentum_vn.md [hunk 8]
```diff
@@ -332,7 +338,7 @@ def sgd_momentum(params, states, hyperparams):
 Let us see how this works in practice.
 -->
 
-*dịch đoạn phía trên*
+Ta hãy xem điều này hoạt động như thế nào trong thực nghiệm.
```
Comment (lkhphuc) [inline chapter_optimization/momentum_vn.md:341]:
```suggestion
Ta hãy xem điều này hoạt động như thế nào trong thực tế.
```

File: chapter_optimization/momentum_vn.md (hunk 3 above)
Comment (lkhphuc) [reply chapter_optimization/momentum_vn.md:360]:
```suggestion
Khi tăng siêu tham số động lượng `momentum` lên 0,9, kích thước mẫu thực tế sẽ tăng lên đáng kể thành $\frac{1}{1 - 0,9} = 10$.
```

File: chapter_optimization/momentum_vn.md [hunk 9]
```diff
@@ -270,7 +270,11 @@ These are two benefits in one.
 To illustrate how weighting behaves for different choices of $\beta$ consider the diagram below.
 -->
 
-*dịch đoạn phía trên*
+Hãy nhớ lại rằng $\mathbf{v}_t = \sum_{\tau = 0}^{t-1} \beta^{\tau} \mathbf{g}_{t-\tau, t-\tau-1}$.
+Tại giới hạn, tổng các số hạng là $\sum_{\tau=0}^\infty \beta^\tau = \frac{1}{1-\beta}$.
+Nói cách khác, thay vì thực hiện bước có kích thước $\eta$ trong GD hoặc SGD, ta thực hiện bước có kích thước $$\frac{\eta}{1-\beta}$, đồng thời hướng hạ gradient nhiều khả năng cũng tốt hơn.
```
Comment (rootonchair) [inline chapter_optimization/momentum_vn.md:275]:
```suggestion
Nói cách khác, thay vì thực hiện bước có kích thước $\eta$ trong GD hoặc SGD, ta thực hiện bước có kích thước $$\frac{\eta}{1-\beta}$, đồng thời hướng hạ gradient nhiều khả năng cũng ổn định hơn.
```

File: chapter_optimization/momentum_vn.md [hunk 10]
```diff
@@ -291,29 +295,31 @@ d2l.plt.legend();
...
-## *dịch tiêu đề phía trên*
+## Các thực nghiệm
 
 <!--
 Let us see how momentum works in practice, i.e., when used within the context of a proper optimizer.
 For this we need a somewhat more scalable implementation.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta hãy xem động lượng hoạt động như thế nào trong thực tế, tức là, khi được sử dụng trong bối cảnh của một bộ tối ưu hóa đích thực.
```
Comment (rootonchair) [inline chapter_optimization/momentum_vn.md:305]:
```suggestion
Chúng ta hãy xem phương pháp động lượng hoạt động như thế nào trong thực tế, tức là, khi được sử dụng trong bối cảnh của một bộ tối ưu hóa đích thực.
```

File: chapter_optimization/momentum_vn.md [hunk 11]
```diff
@@ -291,29 +295,31 @@ d2l.plt.legend();
...
+Để làm điều này, chúng ta cần một cách lập trình có khả năng mở rộng hơn một chút.
```
Comment (rootonchair) [inline chapter_optimization/momentum_vn.md:306]:
```suggestion
Để làm điều này, chúng ta cần một phương pháp lập trình giúp mở rộng dễ dàng hơn.
```

File: chapter_optimization/momentum_vn.md [hunk 12]
```diff
@@ -351,7 +357,8 @@ When we increase the momentum hyperparameter `momentum` to 0.9, it amounts to a
 We reduce the learning rate slightly to $0.01$ to keep matters under control.
 -->
 
-*dịch đoạn phía trên*
+Khi tăng siêu tham số động lượng `momentum` lên 0,9, kích thước mẫu thực tế sẽ tăng lên đáng kể thành $\frac{1}{1 - 0,9} = 10$.
+Chúng tôi giảm tỷ lệ học tập xuống còn $0,01$ để kiểm soát các vấn đề.
```
Comment (rootonchair) [inline chapter_optimization/momentum_vn.md:361]:
```suggestion
Chúng tôi giảm tỷ lệ học tập xuống còn $0,01$ dễ dàng kiểm soát độ hội tụ.
```

File: chapter_optimization/momentum_vn.md [hunk 13]
```diff
@@ -270,7 +270,11 @@ These are two benefits in one.
 To illustrate how weighting behaves for different choices of $\beta$ consider the diagram below.
 -->
 
-*dịch đoạn phía trên*
+Hãy nhớ lại rằng  $\mathbf{v}_t = \sum_{\tau = 0}^{t-1} \beta^{\tau} \mathbf{g}_{t-\tau, t-\tau-1}$.
+Trong giới hạn, các số hạng thêm cộng dồn thành $\sum_{\tau=0}^\infty \beta^\tau = \frac{1}{1-\beta}$.
+Nói cách khác, thay vì thực hiện một bước có kích thước $\eta$ trong GD hoặc SGD, chúng tôi thực hiện một bước có kích thước $$\frac{\eta}{1-\beta}$ đồng thời, xử lý rất nhiều khả năng hành vi hướng xuống tốt hơn.
+Đây là hai lợi ích trong một.
+Để minh họa cách trọng số thực hiện cho các lựa chọn khác nhau của $\beta$, hãy xem xét sơ đồ bên dưới.
```
[the remark this answers is not in the document]
Comment (rootonchair) [reply chapter_optimization/momentum_vn.md:277]: Chắc lúc viết với lúc code họ không thống nhất với nhau nên nó bị khác nhau thôi
